README.md

```diff
@@ -10,60 +10,29 @@ Please see our [Integritee Book](https://book.integritee.network/howto_worker.ht
 To start multiple worker and a node with one simple command: Check out [this README](local-setup/README.md).
```

Comment:
Here we probably want to reference our docker setup as well, since the python local setup is deprecated.
Also, the link for 

> Please see our [Integritee Book](https://book.integritee.network/howto_worker.html) to learn how to build and run this.

is outdated now and returns a 404. The new link is http://oldbook.integritee.network/howto_worker.html, but that information is also outdated. So maybe we should better remove it (or migrate some of the text from https://integritee.atlassian.net/wiki/spaces/INTEGRITEE/pages/294650307/Worker )

Reply:
Good point. I removed the section and added a link to the docker-README.